install_config/syncing_groups_with_ldap.adoc

```diff
@@ -151,6 +151,13 @@ groupUIDNameMapping:
 ----
 ====
 
+[[LDAP-configuration-with-adctive-directory]]
+=== LDAP Configuration with Active Directory
+
+Do not use `objectGUID` as an ID attribute because it is a binary attribute.
```

Comment:
This should state that you should never use any binary attribute and point out objectGUID as an example.